docs/main.css

```diff
@@ -0,0 +1,249 @@
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+/* Header Content */
+header {
+    text-align: center;
+    padding-top: 95px;
+    border: 1px solid #979797;
+    background: radial-gradient(100% 50%, #E9A571 0%, #EE7C5A 82%);
+}
+
+#headline {
+    font-family: "Arial Black";
+    color: #FFFFFF;
+    font-size: 72px;
+    line-height: 101px;

```

Comment:
Some common practice I've seen is pixel dimensions should be a multiplier of 4/8s.

Check out this article https://builttoadapt.io/intro-to-the-8-point-grid-system-d2573cde8632